### ISSUE TITLE
[Bug]: [tae-CN-tae-DN] Duplicated column names in SHOW COLUMNS FROM

### ISSUE THREAD

**Issue by domingozhang** [matrixorigin/matrixone#5820]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

![image](https://user-images.githubusercontent.com/88298673/197102941-6a9e5c1a-811b-4cbf-9030-79ea02e32493.png)

**Comment by daviszhen** [comment]
same reason sa #6041

**Comment by dongdongyang33** [comment]
checking

**Comment by dongdongyang33** [comment]
still checking EOF issue.

**Comment by dongdongyang33** [comment]
EOF issue fixed in #6456
